kt/api-generator/src/main/kotlin/godot/codegen/services/impl/GenerationService.kt

```diff
@@ -754,14 +754,10 @@ class GenerationService(
             FunSpec.builder("new")
                 .addModifiers(KModifier.OVERRIDE)
                 .addParameter("scriptIndex", Int::class)
-                .returns(Boolean::class)
```

Comment:
Please explicitly return `Unit` in these cases. It will be required once we upgrade kotlin poet